src/SFA.DAS.EmployerIncentives.Database/Tables/ApprenticeshipIncentive.sql

```diff
@@ -0,0 +1,16 @@
+﻿CREATE TABLE [incentives].ApprenticeshipIncentive
+(
+	[Id] UNIQUEIDENTIFIER NOT NULL PRIMARY KEY NONCLUSTERED,
+	[AccountId] BIGINT NOT NULL,	
+	[ApprenticeshipId] BIGINT NOT NULL,
+	[FirstName] NVARCHAR(100) NOT NULL,
+	[LastName] NVARCHAR(100) NOT NULL,
+	[DateOfBirth] DATETIME2 NOT NULL,
+	[ULN] BIGINT NOT NULL,	
+	[EmployerType] INT NOT NULL,
+	[PlannedStartDate] DATETIME2 NOT NULL,
+	[IncentiveApplicationApprenticeshipId] UNIQUEIDENTIFIER NOT NULL
+)
+GO
+CREATE UNIQUE CLUSTERED INDEX IX_ApprenticeshipIncentive ON [incentives].[ApprenticeshipIncentive] (AccountId, ApprenticeshipId)
```

Comment:
Actually this unique index doesn't look quite right. The rule we are imposing is a student cannot be funded if he has previously been funded. So I think the unique index should be on the ULN field.

Reply:
Can't be ULN but could be account and ULN...if the learner changes employer before an incentive payment has been made then the new employer can apply.  That rule is currently being reviewed so not sure what the outcome will be.

Reply:
The business rules should be enforced in the code.  Since they may change.  The unique index is because the ApprenticeshipId, AccountId combination should be unique in this table.  Is that not true?